entrypoint.sh

```diff
@@ -1,5 +1,9 @@
-#!/bin/sh
-set -e # exit on error
+#!/bin/bash
```

Comment:
I needed this so I can use arrays in the entrypoint script, because normal `sh` does not support arrays, but `bash` does. I prefer using that over copy/pasting the code twice, which is why I added it.

Reply:
sure but where do you use arrays?

Reply:
I think I found a way to solve it without bash.
```shell
DOCKER_PODMAN_HOSTS="host.docker.internal host.containers.internal"
for host in "${DOCKER_PODMAN_HOSTS[@]}"
do
  echo $host
done
```

Reply:
Nice! In that case using plain `sh` is even better! I did use the array syntax for it, maybe that's why it didn't work for me.